packages/front-end/components/Experiment/EditVariationsForm.tsx

```diff
@@ -54,6 +49,37 @@ const EditVariationsForm: FC<{
         const data = { ...value };
         data.variations = [...data.variations];
 
+        // fix some common bugs
+        if (!isBandit) {
+          const newWeights = [
+            ...data.variations.map((_, i) =>
+              Math.min(
+                Math.max(
+                  data.variationWeights?.[i] ??
+                    1 / data.variations.length ??
```

Comment:
What's the order of operations on `??` Seems like hopefully it's after division and such, in which case I think this makes sense? Might wnat to consider some parentheses for people like me.

Reply:
good call